swagger.yml

```diff
@@ -356,6 +370,12 @@ paths:
             $ref: "#/definitions/ActionResponse"
 
 parameters:
+  roomId:
+    name: roomId
+    in: path
+    description: room identifier
+    required: true
+    type: string
 
   roomName:
```

Comment:
With the path parameter being roomId, we can remove the roomName path parameter.

Reply:
Per the discussion we just had on slack, the roomName parameter definition can remain, but should be updated to be an optional query parameter.  This would be a parameter to GET /api/room